src/frontend/src/sass/_form.scss

```
@@ -57,9 +57,19 @@
   margin: 0;
 }
 
+.search-container-wrapper {
+  display: flex;
+  align-items: center;
+  margin-top: 15px;
```

This `1em` should be 16px. We want to avoid pixel values.
```suggestion
  margin-top: 1em;
```